documentation/docs/src/user-guide/ibc.md

```diff
@@ -0,0 +1,252 @@
+# Using IBC with Namada
```

Comment:
I think that this doc has twos parts:
1. Setup Hermes (without the script)
2. Local testing with the script

I suggest that it should be separated clearly. How about making the second part an example of setting up Hermes (though the config will be created)?

### Setup Hermes for Namada
  1. Make an account for Hermes on each instance
  2. Make Hermes config.toml
  3. Set up a channel (also a client and a connection) between two instances
  4. Run Hermes to relay packets between instances

  (These operations are to set up Hermes for any Namada instances. And they are not specific for the script `setup-namada.sh`.)
 
### Setup Hermes for local Namada instances
...